easybuild/easyconfigs/v/velocyto.py/velocyto.py-0.17.17-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'velocyto.py'
```

Comment:
I think using `velocyto` as name makes more sense, the Python aspect is already reflected in the `versionsuffix`; see also https://pypi.org/project/velocyto/